### PR TITLE
AMP-8 login page

### PR DESCRIPTION
**Pull request by AniketRaj314**
TODO:
- Send Request to API for verifying login
- Add Form Validation
- Improve UI

### CODE REVIEW

**Comment by ksdfg** [comment]
@vercel[bot] eeeek why failed

**Comment by akhilnarang** [comment]
image

**Comment by AniketRaj314** [comment]
Ahh, I see. Lemme update that fir. Thanks. Where did you find the logs though?

**Comment by akhilnarang** [comment]
My vercel account, so it shows for me :3
Will see if I can create a combined one.

**Comment by ksdfg** [comment]
image

incomplete page, or rekt CSS?

File: src/pages/login/login.page.jsx [hunk 1]
```diff
@@ -35,14 +35,12 @@ class Login extends Component {
...
-                    throw Error(response.json());
-                }
-                return response.json();
+            .then(async response => {
+                const json = await response.json();
+                return response.ok ? json : Promise.reject(json);
             })
             .then(token_object => this.setState({ user_auth_token: token_object.access_token }, this.getUser))
-            .catch(error => toast.error(`Error 😓: ${error.message}`, {
+            .catch(error => toast.error(`Error 😓: ${error['detail']}`, {
```
Comment (AniketRaj314) [inline src/pages/login/login.page.jsx]: LGTM, just can we make it error.detail for consistency purpose?

**Review by AniketRaj314 — COMMENTED** [review]
RIP I wrote same code, but forgot to put async before response in the params.

**Review by akhilnarang — APPROVED** [review]

**Review by adityavdesai — APPROVED** [review]

**Pull request merged**